consul/config_templates/consul.yaml

```diff
@@ -575,24 +546,31 @@ spec:
             periodSeconds: 2
             successThreshold: 1
             timeoutSeconds: 5
-
+          resources:
```

Comment:
Same as below

Reply:
Sorry, I'm not quite sure I understand this either.

Reply:
Same as below